<launch>
    <group ns = "rqt_reconfigure_tutorial">
        <node pkg="rqt_reconfigure_tutorial" type="reconfigure_node" name="reconfigure_node" output="screen">
            <rosparam file="$(find rqt_reconfigure_tutorial)/param/sample.yaml" command="load"/>
        </node>
    </group>

    <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="rqt_reconfigure"/>
</launch>
